Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM jekyll/jekyll
```

Comment:
I don't think we need to extend jekyll image, since we are not doing anything custom.

It should be possible to do everything with provided jekyll images?

Reply:
Yes, you're right. Thanks for your input!